.github/workflows/install_script.yml

```
@@ -18,11 +18,12 @@ on:
 
 jobs:
   run_install_script:
-    runs-on: ubuntu-latest
     strategy: 
       matrix:
         branch: ['develop', 'stable'] # 'develop' can be anything that's not 'stable'. This triggers an if statement in ubuntu.sh
         build_hdf5: ['NO', 'hdf5-1_12_0']
+        os: ['ubuntu-latest', 'ubuntu-20.04']
```

I believe we should consider using a specific version of Ubuntu as the latest regularly updated version, and soon we will have Ubuntu 24.04. 

```suggestion
        os: ['ubuntu-22.04', 'ubuntu-20.04']
```